packages/actions/src/eth/ethSendRawTransaction.spec.ts

```
@@ -0,0 +1,146 @@
+    expect(accountState.balance).toBe(value)
+  })
+
+  it.todo('should handle a legacy transaction', async () => {
+    const client = createTevmNode()
+    const handler = ethSendRawTransactionHandler(client)
+
+    const tx = TransactionFactory.fromTxData({
+      nonce: '0x00',
+      gasPrice: '0x09184e72a000',
+      gasLimit: '0x2710',
+      to: createAddress('0x' + '42'.repeat(20)),
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)